README.md

```diff
@@ -158,12 +171,12 @@ If you'd like to change, patch or alter files or behavior of `docker-mailserver`
 
 #### Updating `docker-mailserver`
 
-Make sure to read the [CHANGELOG](https://github.com/docker-mailserver/docker-mailserver/blob/master/CHANGELOG.md) before updating to new versions, to be prepared for possible breaking changes.
+**Make sure to read the [CHANGELOG](https://github.com/docker-mailserver/docker-mailserver/blob/master/CHANGELOG.md)** before updating to new versions, to be prepared for possible breaking changes.
 
-``` BASH
-docker-compose pull
-docker-compose down
-docker-compose up -d mailserver
+``` CONSOLE
+$ docker-compose pull
+$ docker-compose down
+$ docker-compose up -d mailserver
```

Comment:
1. Whats the difference between using BASH / CONSOLE here? I've never seen CONSOLE used before.
2. Adding `$` here makes easy copy/paste harder.

Reply:
I think it might have been to distinguish between commands and output, it's just for syntax highlighting, so bash is probably fine too.

Reply:
`CONSOLE` is for displaying "an interactive shell", i.e. when you want to show to people that these commands are to be run in a shell. I disagree that this makes copy-pasting harder - someone who cannot copy a line properly (be it here or elsewhere in the docs) should probably not operate a mail server :D

Reply:
> someone who cannot copy a line properly

I didn't say it's not possible or so, but it makes it harder. You cannot copy / paste these three lines at once. You'll have to paste them somewhere else and remove the `$` or you'll have to copy each line alone.
No big deal in the end, but I would remove them. Leave it as it is if you disagree, no reason to discuss it to death 😉

Reply:
I will revert to `BASH` in this case, as there is no output from the individual commands. The rendering with `CONSOLE` shows a clear distinction between the command and the output, which is preferred in some situations - but not in this one particularly.